<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Addressbook</class>
 <widget class="QMainWindow" name="Addressbook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>652</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>439</width>
    <height>652</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>439</width>
    <height>652</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>楷体</family>
    <pointsize>10</pointsize>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset theme="user-home">
    <normaloff>../build-sqlite-Desktop_Qt_6_2_4_MinGW_64_bit-Debug/a.ico</normaloff>
    <normalon>../build-sqlite-Desktop_Qt_6_2_4_MinGW_64_bit-Debug/a.ico</normalon>
    <disabledoff>../build-sqlite-Desktop_Qt_6_2_4_MinGW_64_bit-Debug/a.ico</disabledoff>
    <disabledon>../build-sqlite-Desktop_Qt_6_2_4_MinGW_64_bit-Debug/a.ico</disabledon>
    <activeoff>../build-sqlite-Desktop_Qt_6_2_4_MinGW_64_bit-Debug/a.ico</activeoff>
    <activeon>../build-sqlite-Desktop_Qt_6_2_4_MinGW_64_bit-Debug/a.ico</activeon>
    <selectedon>../build-sqlite-Desktop_Qt_6_2_4_MinGW_64_bit-Debug/a.ico</selectedon>../build-sqlite-Desktop_Qt_6_2_4_MinGW_64_bit-Debug/a.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(198, 184, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>61</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>550</y>
      <width>61</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>441</width>
      <height>451</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>441</width>
      <height>451</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>441</width>
      <height>451</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>550</y>
      <width>41</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="digitCount">
     <number>8</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>610</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>334</x>
      <y>550</y>
      <width>41</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
